bazel/v8.patch

```diff
@@ -1,15 +1,6 @@
-# 1. Use already imported python dependencies.
-# 2. Fix the include path for //external:zlib.
-# 3. Add support for --define=no_debug_info=1.
-# 4. Disable pointer compression (limits the maximum number of WasmVMs).
-# 5. Don't expose Wasm C API (only Wasm C++ API).
-# 6. Fix cross-compilation (https://crrev.com/c/3735165).
-# 7. Fix build errors in SIMD IndexOf/includes (https://crrev.com/c/3749192).
-# 8. Fix build on arm64.
-# 9. Fix build on older versions of Linux.
-# 10. Fix MemoryAllocator::PartialFreeMemory() which shouldn't try to change permissions of RWX pages,
-#     mainly affecting macOS on Apple silicon (https://crrev.com/c/3700352). This can be removed
-#     when we adopt 10.5 or higher (https://github.com/envoyproxy/envoy/issues/23258).
+# 1. Use already imported python dependencies and disable pointer compression (limits the maximum number of WasmVMs).
```

Comment:
Nit: Those are 2 separate changes, so it's weird to combine them here.

Reply:
I'll update the patch hence the separation is clear.